src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -255,6 +256,10 @@ export default Vue.extend({
           this.hasLoadMapError = true;
         });
     },
+    addEmptyCard(dataDriven) {
+      const name = dataDriven ? "Data driven" : "Design";
+      this.addCard(name, null, null, "", dataDriven);
```

Comment:
I think it's preferable to use `null` here for the method, for consistency, rather than empty string